lib/chai/core/assertions.js

```diff
@@ -529,16 +529,35 @@ module.exports = function (chai, _) {
   function assertAbove (n, msg) {
     if (msg) flag(this, 'message', msg);
     var obj = flag(this, 'object')
-      , doLength = flag(this, 'doLength');
+      , doLength = flag(this, 'doLength')
+      , objType = _.type(obj)
+      , nType = _.type(n)
+      , msgPrefix = (msg) ? msg + ': ' : '';
 
     if (doLength) {
       new Assertion(obj, msg).to.have.property('length');
     } else {
-      new Assertion(obj, msg).is.a('number');
+      // Since utils/expectTypes is missing from 4.x.x and no better way to assert multiple types
```

Comment:
I could not find a better way to assert multiple types, on the original PR `expectTypes` was suggested, but it's no longer present in the utils